src/util/delay.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-env jest */
+
+import delay from 'src/util/delay'
+jest.useFakeTimers()
+
+describe('delay tests', () => {
+    test('should return with the given delay', () => {
+        expect.assertions(2)
+        delay(1000)
+        expect(setTimeout.mock.calls.length).toBe(1)
+        expect(setTimeout.mock.calls[0][1]).toBe(1000)
+    })
+})
```

Comment:
The motive of the tests wasn't to test whether or not the promise is resolved, it was to check whether the specified timeout was called. The test can be changes, but I felt this implementation was better for this particular module.